translations/base-fr.yaml

```diff
@@ -710,11 +711,11 @@ settings:
         language:
             title: Langue
             description: >-
-                Change la langue. Toutes les traductions sont contribuées par les utilisateurs et peuvent être incomplètes !
+                Change la langue. Toutes les traductions sont réalisées par les utilisateurs et peuvent incomplètes !
```

Comment:
I think "contribuer" is better than "réaliser" here, but either way this sentence accidentally a verb

Reply:
the first time i see "contribuer" in passive form I was stuck. I know the sentence is correct but it feel weird to me.

Reply:
I asked on a frech IRC channel:

```
Nah : Bonjour. Petite question. Vous préférez laquelle des 2 phrases ?
Nah : 1) Toutes les traductions sont contribuées par les utilisateurs et peuvent être incomplètes !
Nah : 2) Toutes les traductions sont réalisées par les utilisateurs et peuvent être incomplètes !
Map : "sont contribuées" ?
Eso : effectuées
Tej : sont contribuées ça n'a pas de sens
Tej : les traductions sont une contribution des utilisateurs & peuvent être incomplètes
Eso : comme ça encore ça va
Map : Les utilisateurs ont contribué à toutes les traductions, elles peuvent être incomplètes
```
And the three who answer find "sont contribuées par" weird. But I can't chose between mine or  Tej's or Map's.

Reply:
Map's is misleading: "ont contribué à" denotes partial contribution, not complete autorship.

I think I like Tej's best.